proto/steward/v4/steward.proto

```diff
@@ -48,11 +48,17 @@ message ScheduleRequest {
         CellarV2 cellar_v2 = 5;
         CellarV2_2 cellar_v2_2 = 6;
     }
+    // The ID of the chain on which the target Cellar resides
+    uint64 chain_id = 7;
```

Comment:
I guess we're going to handle adding CellarV2_5 in main parity?

Reply:
yes